### ISSUE TITLE
[Financial] ENABLE the trend-alignment filter on demo — shipped default-off (#48); counter-trend = 93% of loss, 07-13 confirms

### ISSUE THREAD

**Issue by magedzamzam** [magedzamzam/beacon-trader#72]
## Summary
The trend-alignment entry filter (#48) was shipped **DEFAULT-OFF** behind an A/B flag (`entry_filters.trend_alignment.enabled`, seeded disabled). The live 2026-07-13 dump has **no `entry_filters` config row** and counter-trend trades are still being placed. The evidence for the filter is now materially **stronger** than when #48 was written, and it is the single highest-EV lever on the board — currently dormant.

**Action: enable the filter on demo (acct#5)** — `entry_filters.trend_alignment {enabled:true, timeframe:"4h", ema_period:200, mode:"skip"}` (or `mode:"desize", desize_factor:0.25` if you prefer a softer A/B). Then track the aligned-vs-counter split as a first-class metric.

## Evidence (2026-07-13 dump, n=135 labelled trades, acct#5, ledger PnL)
Direction classified by price vs 4h EMA200 at signal time (`signal_features`):

| Population | n | win% | net AED | exp/trade |
|---|--:|--:|--:|--:|
| Trend-ALIGNED | 68 | 56% | −4,637 | −68 |
| **COUNTER-trend** | 67 | 34% | **−60,319** | **−900** |

**Counter-trend entries hold −60,319 of the −64,956 ledger loss = 93%.** Aligned entries are ≈ breakeven.

**Within the keeper channels (#7/#9/#12) — additive, not a substitute for disables:**
| Bucket | n | net AED | win% |
|---|--:|--:|--:|
| Keepers, aligned-only | 18 | **+18,336** | 72% |
| Keepers, counter-trend | 21 | −17,014 | 33% |

**Prospective confirmation on 07-13 (Monday, the newest trading day):** aligned n=8 → **+1,519 (88% win)**; counter n=9 → **−15,347 (33% win)**. The filter would have skipped the 9 counter-trend trades that drove the day's ledger loss.

## Caveat (unchanged from #48)
Validated over a single ~5-day *bearish* 4h regime (aligned currently manifests as SELL). The *mechanism* — don't fight the higher-TF trend — is regime-agnostic, but re-verify when the 4h trend flips bullish. This is a filter on the mechanism, **not** a standing "sell gold" bias. Keep it on demo behind the A/B flag and watch the split.

Refs #48. Broker-truth cross-check and full method in today's quant report.

**Comment by magedzamzam** [comment]
Done in 9212507 (pushed to main).

## Enabled the filter (reversible, platform-controlled)
`seed.py` now seeds `entry_filters.trend_alignment` **enabled** with the exact params from the issue — `{enabled:true, timeframe:"4h", ema_period:200, mode:"skip"}`. The live dump had **no entry_filters row**, so on the next API start the seed creates it enabled and the executor begins skipping counter-trend entries.

Design guardrails kept intact:
- **Reversible from the platform** — the Risk page's *Trend-alignment entry filter* card (PUT `/entry-filters/config`) toggles it off or switches to `desize` live. Nothing is hardcoded.
- **`setdefault`** — re-seeding never clobbers an operator who later tunes it.
- **Library default stays OFF** — the opt-in A/B contract and `test_default_is_off_and_fail_open` are unchanged. This seeds the *deployment's* rollout value, not a standing "sell" bias. Per the caveat, re-verify when the 4h trend flips bullish (I kept `mode:skip`; flip to `desize · 0.25` from the Risk page if you want the softer A/B).

## Tracked the aligned-vs-counter split as a first-class metric
- `alignment_from_features()` — pure classifier reading the 4h EMA200 `above` flag from persisted `signal_features` (the **same** definition the live filter gates on).
- `trend_alignment_outcome_report()` — aligned vs counter win-rate, net PnL, expectancy, overall + per channel, with Beta-Binomial credible intervals.
- **GET `/analytics/trend-alignment`** — reads the live `entry_filters` config so the report always matches the timeframe/EMA the filter actually runs on.
- New **Trend alignment vs outcome** card on the Analytics page.

**Validation:** 2 new pure tests for the classifier (classification + fail-open on missing EMA/tf). Full pure suite **137 passed, 0 failed** (CI-only redis/sqlalchemy modules skip on the dev box). Frontend verified by eye (no npm here — builds in Docker/CI).

## Deploy (on beacon-vnic)
```
cd ~/beacon-trader && git pull
docker compose build api executor monitor telegram frontend
docker compose up -d api executor monitor telegram frontend
```
After `api` restarts, confirm on the **Risk** page that the card shows *on · skip*, or:
```
curl -s localhost:8000/entry-filters/config -H "authorization: Bearer $TOKEN" | jq .trend_alignment
```
**One caveat:** `setdefault` only *creates* the row — if the demo DB already has an `entry_filters` row with the filter disabled (the dump indicates it doesn't), the seed won't flip it; just toggle it on once from the Risk page. Watch the split on the new Analytics card and re-verify when the 4h regime turns bullish.
